pages/linux/uname.md

```diff
@@ -0,0 +1,36 @@
+# uname
+
+> Uname prints information about the machine and operating system it is run on.
+> More information: <https://www.gnu.org/software/coreutils/manual/html_node/uname-invocation.html>.
+
+- Print all information:
+
+`uname --all`
+
+- Print the kernel name:
```

Comment:
```suggestion
- Print a kernel name:
```

Reply:
`A kernel` is very ambiguous, because you could ask "which kernel". `The kernel` on the other side clearly means `the current kernel`. Same for your other `the → a` suggestions.

Reply:
On the other side I don't talk about kernel used in my PC, I talk about any kernel that is current to any computer. From this point of view I suggested `a`. :)

Reply:
All suggestions are updated.

Reply:
`Current kernel` is also good.